packages/protocol/contracts-0.8/governance/Validators.sol

```diff
@@ -611,7 +613,6 @@ contract Validators is
    * @param validatorAccount The validator to deaffiliate from their affiliated validator group.
    */
   function forceDeaffiliateIfValidator(address validatorAccount) external nonReentrant onlySlasher {
-    allowOnlyL1();
```

Comment:
TODO probably needs to stay removed https://github.com/celo-org/celo-monorepo/pull/11225